.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
   "recommendations": [
     "dbaeumer.vscode-eslint",
-    "flowtype.flow-for-vscode",
```

Comment:
Going to (eventually) replace Flow with TS.
This will probably happen during the Emotion 9 -> 10 migration